src/Service/Kms/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 - Added `me-central-1`, `me-central-1-fips`, `ap-south-2-fips`, `eu-south-2-fips` and `eu-central-2-fips` regions
 - AWS api-change: Added `eu-central-2` region
+- AWS api-change: Added support for the SM2 KeySpec in China Partition Regions
```

Comment:
None of these change log entries are relevant, and we're missing a note for the new eu region.